building.sk

```diff
@@ -622,3 +622,23 @@ village and pillage update:
 	[any] stone slab¦s = stone slab, smooth stone slab, granite slab, diorite slab, andesite slab, polished granite slab, polished diorite slab, polished andesite slab, mossy stone brick slab, mossy cobblestone slab
 	[any] sandstone slab¦s = smooth sandstone slab, smooth red sandstone slab, cut sandstone slab, cut red sandstone slab
 
+nether update:
+	minecraft version = 1.16 or newer
+
+	ancient debris = minecraft:ancient_debris
+	basalt = minecraft:basalt
+	netherite block¦s = minecraft:netherite_block
+	soul soil¦s = minecraft:soul_soil
+	warped wart block = minecraft:warped_wart_block
```

Comment:
This block should be plural `block¦s` like the others. Also, I know the name you have here is technically the completely correct full name, but it might be good to add `nether` as an optional word in the middle so people can write `warped nether wart block¦s`. I imagine some people will make that mistake otherwise.